rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly-terraform.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: ASSEMBLY
+[id="rosa-sts-creating-a-cluster-quickly-terraform"]
+= Creating a ROSA cluster with STS using the default options with Terraform
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-sts-creating-a-cluster-quickly-terraform
+
+toc::[]
+
+[NOTE]
+====
+If you are looking for a quickstart guide for ROSA, see xref:../rosa_getting_started/rosa-quickstart-guide-ui.adoc#rosa-quickstart-guide-ui[{product-title} quickstart guide].
+====
+
+Create a {product-title} (ROSA) cluster quickly by using a Terraform cluster template that is configured with the default cluster options.
+
+If you want to install ROSA clusters with the default options by using the CLI or {cluster-manager-url}, see xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc#rosa-sts-creating-a-cluster-quickly-ocm_rosa-sts-creating-a-cluster-quickly[Creating a ROSA cluster with STS using the default options]. For steps to deploy a ROSA cluster by using `manual` mode or with customizations, see xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.adoc#rosa-sts-creating-cluster-using-customizations_rosa-sts-creating-a-cluster-with-customizations[Creating a cluster using customizations].
+
+This cluster creation process uses a Terraform configuration that prepares a ROSA Classic STS cluster with the following resources:
+
+* an OIDC provider with a managed oidc-config,
+* prerequisite Operator roles with policies,
+* IAM account roles with policies,
+* and all other AWS resources required to create a ROSA cluster.
+
```

Comment:
The list above looks somewhat strange to me with the combo of bullets and commas. Not sure if we have used this format before, but maybe we have?